### DISCUSSION TITLE
Feed showing incorrect content

### DISCUSSION THREAD

**Discussion by tazjin** [community/community#27709] [Other Feature Feedback, Questions, & Ideas]
The new feed keeps showing me content from people I don't know / care about / don't follow. This is not useful compared to the previous feed where I could opt in to things that are useful to me. The effect is that I don't read it anymore and probably miss stuff I'd have liked to know about otherwise.

It does seem like a bug to me but on the off-chance that Github is trying to be Facebook here: Please don't. Give me an option to disable "smart" features - you don't know what I want.

**Comment by MrSarno** [comment]
The new feed is not replacing the Following feed; it wouldn't be fantastically helpful if it, too, was limited in scope to users you follow. :)

Using the filter to remove the following may help;

- Sponsors
- Stars
- Repositories
- Follows
- Recommendations

That said, it's possible the new feed just isn't for you. Which is fine!

Not every feature needs to appeal to every user. The Following feed is still there if you prefer that one.

**Comment by tazjin** [comment]
> Not every feature needs to appeal to every user. The Following feed is still there if you prefer that one.

I didn't notice this before you said it, and tried it for a few days. The Following feed is working normally, but now the problem is that Github occasionally just switches you back to the new "spammy" feed.

It feels like when Twitter dropped chronological timelines by default, but still made them available through a dropdown menu that occasionally reset itself. I eventually stopped using Twitter.

My solution for now is to just block the feed stuff completely in uBlock, it's more important to not get spammed. I'll have to figure out some kind of feed-following mechanism for important projects that is independent of Github UI.
